index.html

```diff
@@ -8,50 +8,266 @@
-                <a href="# " rel="noreferrer"><img src="./assets/angellist.svg" alt="Angellist icon" /></a>
-            </li>
-        </ul>
     </main>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/avocode-mobile-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Tonic</h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">CANBOY</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Back End Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/availability-mobile-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Multi-Post Stories</h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">Facebook</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Full Stack Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">Ruby on rails</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/gym-fit-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Tonic</h1>
```

Comment:
- [ ] There should be only one `<h1>` tag on a page for semantic and SEO reasons. Kindly replace `<h1>` tag other than the one used for your name in the first section with `<hx>` tags (h2. .... h6).